templates/cronjob.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: platform
+  name: aws-credentials-updater
+
+---
+
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  namespace: platform
+  name: secret-edit
+rules:
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["get", "watch", "list", "update"]
+
+---
+
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  namespace: platform
+  name: aws-credentials-updater-rolebinding
+subjects:
+- kind: ServiceAccount
+  name: aws-credentials-updater
+roleRef:
+  kind: Role
+  name: secret-edit
+  apiGroup: rbac.authorization.k8s.io
+
+---
+
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  namespace: platform
+  name: rotate-keys
+spec:
+  schedule: "30 03 01 */3 *"  # It takes a Cron format string, as schedule time of its jobs to be created and executed (in our case the job Will run every three months at 03:30 on the 1st of Jan,Apr,Jul and Oct).
+
+).
```

Comment:
Wrong content?

Reply:
what do you expect ...

Reply:
@doulba, I think @kevinleturc is talking about the extra blank line and the `).` characters.

Reply:
sorry